src/main/haskell/kore/src/Kore/Reflect.hs

```diff
@@ -0,0 +1,386 @@
+{-# LANGUAGE DefaultSignatures #-}
```

Comment:
`LANGUAGE DefaultSignatures` and `LANGUAGE TypeOperators` should be enabled project-wide.

Reply:
Ok, I added them.